python/pacemaker/_cts/clustermanager.py

```diff
@@ -878,11 +878,15 @@ def StandbyStatus(self, node):
 
     # status == "on" : Enter Standby mode
     # status == "off": Enter Active mode
-    def SetStandbyMode(self, node, status):
+    def set_standby_mode(self, node, status):
         current_status = self.StandbyStatus(node)
+
+        if current_status == status:
+            return True
```

Comment:
FYI this could possibly miss some cases since the command accepts a variety of synonyms for on and off, but existing callers do use on and off, and it would still work anyway so no big deal

It would be simpler to take a boolean argument and map it to on/off